.setup/bin/populate_office_hours_queue.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+import argparse
+from sqlalchemy import create_engine, Table, MetaData, select, update, insert
+import random
+import os
+from datetime import datetime, timedelta
+
+DB_HOST = "localhost"
```

Comment:
Is this intended to be run from inside or outside the VM?
A lot of the scripts in .setup are expected to be executing from inside so they can access the configuration data that gets generated with the first VM up

so from inside the VM you could do something like
`python3 GIT_CHECKOUT/Submitty/.setup/bin/<YOUR_SCRIPT.py>` and in python access the `$SUBMITTY_INSTALL_DIR/.setup/submitty_conf.json` file

which contains whatever the user configured their DB's host/port/username/pswd to be

Reply:
This script is intended to be run inside the VM. I just hardcoded the variables from setup_sample_course as I was trying to have a semi-functional prototype while I coded it up, I will add a note at the top of the script saying its usage. I will also grab the information from the file you suggested.